10-inference.Rmd

```diff
@@ -210,41 +210,29 @@ the mean of the sample proportions.
 sample_estimates %>%
   summarise(mean = mean(sample_proportion))
 ```
-We notice that the sample proportions are centred around the population proportion value, `r round(mean(sample_estimates$sample_proportion), 2)`! In general, the mean of the distribution of $\hat{p}$ should be equal to $p$, which is good because that means the sample proportion is neither an overestimate nor an underestimate of the population proportion.
+We notice that the sample proportions are centred around the population proportion value, `r round(mean(sample_estimates$sample_proportion), 3)`! In general, the mean of the distribution of $\hat{p}$ should be equal to $p$, which is good because that means the sample proportion is neither an overestimate nor an underestimate of the population proportion.
```

Comment:
I would expand a bit to clarify the statement that "sample proportion is neither an overestimate or underestimate"

Reply:
Good point! I added the line: "Any *individual* sample may give you a 
sample proportion that is larger or smaller than the population proportion, but when we repeat the sampling process many times (as we did in this example), there is no tendency towards over or underestimating." Hopefully that is better!